app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> int:
+    return sum(player.get_rating() for player in players)
```

Comment:
The `calculate_team_total_rating` function returns the sum of ratings as an integer. If you intend to return an average rating, you should divide the sum by the number of players. Additionally, ensure that the division is done using floating-point division to avoid truncation to an integer.